src/olympia/abuse/models.py

```diff
@@ -853,34 +844,40 @@ def get_action_helper_class(cls, decision_action):
             DECISION_ACTIONS.AMO_ESCALATE_ADDON: CinderActionEscalateAddon,
             DECISION_ACTIONS.AMO_DELETE_COLLECTION: CinderActionDeleteCollection,
             DECISION_ACTIONS.AMO_DELETE_RATING: CinderActionDeleteRating,
-            DECISION_ACTIONS.AMO_APPROVE: CinderActionApproveInitialDecision,
+            DECISION_ACTIONS.AMO_APPROVE: CinderActionApproveNoAction,
+            DECISION_ACTIONS.AMO_APPROVE_VERSION: CinderActionApproveInitialDecision,
```

Comment:
That's not new, but `CinderActionApproveInitialDecision` is a confusing name when it comes to proactive decisions...